hknweb/templates/landing/home.html

```diff
@@ -1,5 +1,35 @@
 {% extends "base.html" %}
 {% load static from staticfiles %}
+<html>
+    <head>
+
+        <!-- Base CSS -->
+        <link rel="stylesheet" href="{% static "css/home.css" %}">
+
+        <!-- Fonts -->
+        <link href="https://fonts.googleapis.com/css?family=Montserrat:400,500,600,700,800,900|Open+Sans|PT+Sans:700|Roboto:400,500,700,900|Source+Sans+Pro:300,400,600,700" rel="stylesheet">
+
+        <!-- Materialize -->
+        <!-- Compiled and minified CSS -->
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
+        <!-- Compiled and minified JavaScript -->
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
```

Comment:
Is materialize actually used on this page?

Reply:
idk... I don't remember adding this

Reply:
So should I remove it?